lib/generated/intl/messages_en.dart

```diff
@@ -22,22 +22,41 @@ class MessageLookup extends MessageLookupByLibrary {
   final messages = _notInlinedMessages(_notInlinedMessages);
   static _notInlinedMessages(_) => <String, Function> {
     "aboutApp" : MessageLookupByLibrary.simpleMessage("About"),
-    "apiConnectionException" : MessageLookupByLibrary.simpleMessage("No internet connection"),
     "apiInvalidUrlException" : MessageLookupByLibrary.simpleMessage("Resource not found"),
     "apiServerException" : MessageLookupByLibrary.simpleMessage("Internal server error"),
     "apiUnauthorizedException" : MessageLookupByLibrary.simpleMessage("Authorization error"),
     "apiUnknownException" : MessageLookupByLibrary.simpleMessage("Unknown error"),
     "appInfoFeatureDescription" : MessageLookupByLibrary.simpleMessage("The latest news from the world"),
     "appInfoTechnicalDescription" : MessageLookupByLibrary.simpleMessage("Application fully written in Flutter"),
+    "createAccount" : MessageLookupByLibrary.simpleMessage("Create account"),
     "darkMode" : MessageLookupByLibrary.simpleMessage("Dark mode"),
+    "email" : MessageLookupByLibrary.simpleMessage("E-mail"),
     "emailed" : MessageLookupByLibrary.simpleMessage("E-mailed"),
+    "emptyFieldError" : MessageLookupByLibrary.simpleMessage("This field cannot be empty"),
+    "firebaseEmailAlreadyInUseError" : MessageLookupByLibrary.simpleMessage("E-mail address already in use"),
```

Comment:
ogólnie unikałbym przedrostka firebase w nazwach stringsów - przecież one nie wiedzą, że to błąd z firebase :D no i przy choćby prostej zmianie np. walidacji lokalnej e-maila regexem nie trzeba będzie zmieniać klucza komunikatu błędu :)